### ISSUE TITLE
Typescript import

### ISSUE THREAD

**Issue by Dragomir-Ivanov** [MikeMcl/decimal.js-light#5]
Hi there,
I have issue importing this library to angular-cli project.
Import like:
```
import { Decimal } from 'decimal.js-light';
```
code:
```
    let d = new Decimal(1.23);
    console.log(d.toJSON());
```

I get the error:
```
_WEBPACK_IMPORTED_MODULE_4_decimal_js_light_decimal__.Decimal is not a constructor
    at ClientsComponent.webpackHotUpdate.../../../../../src/app/ftech/clients/clients.component.ts.ClientsComponent.onRefresh (clients.component.ts:89)
    at Object.eval [as handleEvent] (ClientsComponent.ngfactory.js:38)
    at handleEvent (
````

Typescript 2.4.2, code completion works fine, i.e. recognizes `decimal.d.ts` file.

**Comment by MikeMcl** [comment]
Hi,

Does `import Decimal from 'decimal.js-light';` work?

Does `import { Decimal } from 'decimal.js-light';` work with the version of `decimal.js` attached to this comment?

[decimal.js](https://github.com/MikeMcl/decimal.js-light/files/1288688/decimal.js.txt)

**Comment by Dragomir-Ivanov** [comment]
@MikeMcl Hi Mike, 
`import Decimal from 'decimal.js-light';` doesn't work with 2.2.4, however
`import { Decimal } from 'decimal.js-light';` with the new file given works as expected. Thank you very much for the fast reaction. Waiting for the next version eagerly :)

**Comment by MikeMcl** [comment]
Fixed in v2.2.5.
You're welcome.
